### PR TITLE
enigmail: add a warning about default created gpg key by enigmail

### PR DESCRIPTION
**Pull request by fepitre**
Related to https://github.com/QubesOS/qubes-issues/issues/5639

### CODE REVIEW

**Comment by marmarek** [comment]
Travis detected typo: `defaut`

File: user/security-in-qubes/split-gpg.md [hunk 1]
```diff
@@ -156,6 +156,10 @@ It is recommended to set up and use `/usr/bin/qubes-gpg-client-wrapper`, as disc
 
 ![tb-enigmail-split-gpg-settings-2.png](/attachment/wiki/SplitGpg/tb-enigmail-split-gpg-settings-2.png)
 
+**Warning:** By default, Enigmail could generate a default GPG key in `work` associated with the newly created Thunderbird account. Generally, it corresponds to the email used in
+`work-gpg` associated to your private key. In consequence, you will obtain `gpg -K` in `work` being non-empty but it _does not_ correspond to your private key in `work-gpg`.
```
Comment (marmarek) [inline user/security-in-qubes/split-gpg.md]: Not everyone knows gpg options. Better reword it to something like "In consequence, a new, separate private key will be stored in `work`, ...`

Comment (fepitre) [reply user/security-in-qubes/split-gpg.md]: > Do you remember in what order have you configured this? Does it also happen if you set `qubes-gpg-client-wrapper` in TB settings _before_ adding an account?

I don't remember but I can easily redo the test.

> If not, maybe you should also mention that it's important to set `qubes-gpg-client-wrapper` before adding an account?
